Packs/HPE_Service_Manager/Integrations/integration-HPE_Service_Manager.yml

```diff
@@ -1,32 +1,40 @@
 commonfields:
-  id: Service Manager
+  id: Micro Focus Service Manager
```

Comment:
won't this break BC?

Reply:
it will, @mdrobniu please revert id and name